hack/dep-prune.sh

```diff
@@ -3,3 +3,5 @@ for file in $(find vendor/ -name "*_test.go"); do rm ${file}; done
 rm -rf vendor/github.com/golang/glog
 mkdir -p vendor/github.com/golang/
 ln -s ../../../pkg/staging/glog/ vendor/github.com/golang/glog
+#create link of client-go from staging
+ln -s ../../staging/src/kubevirt.io/client-go vendor/kubevirt.io/client-go
```

Comment:
you should do it via go modules, insert under the go.mod and run `make deps-update`
```
replace kubevirt.io/client-go => ./staging/src/kubevirt.io/client-go
```

Reply:
gomodules support symlinking? nice. We should do the same with glog too.

Reply:
@rmohr In the end, it does not support symbolic links, to introduce correct vendoring with symbolic links we will need logic similar to k8s one
1. update vendor - https://github.com/kubernetes/kubernetes/blob/master/hack/update-vendor.sh
2. update bazel files - https://github.com/kubernetes/kubernetes/blob/master/hack/update-bazel.sh

Gazelle also does not like symbolic links under vendor, because in this case, it will generate `importpath` under BUILD file `kubevirt.io/kubevirt/staging/src/kubevirt.io/client-go/kubecli`

Reply:
Sounds good :+1: